test/12-device-config.spec.ts

```diff
@@ -402,46 +561,26 @@ describe('Device Backend Config', () => {
 				),
 			).to.equal(true);
 
-			// @ts-ignore accessing private value
 			await deviceConfig.setBootConfig(configFsBackend, target);
 			expect(logSpy).to.be.calledTwice;
 			expect(logSpy.getCall(1).args[2]).to.equal('Apply boot config success');
-			expect(fsUtils.writeAndSyncFile).to.be.calledWith(
-				'test/data/mnt/boot/configfs.json',
+			expect(await fs.readFile(configFsJson, 'utf-8')).to.equal(
 				'{"ssdt":["spidev1.0"]}',
 			);
 		});
 
 		it('should correctly load the configfs.json file', async () => {
 			stub(fsUtils, 'exec').resolves();
-			stub(fsUtils, 'exists').resolves(true);
-			stub(fs, 'mkdir').resolves();
-			stub(fs, 'readdir').resolves([]);
-			stub(fs, 'readFile').callsFake((file) => {
-				if (file === 'test/data/mnt/boot/configfs.json') {
-					return Promise.resolve(
-						JSON.stringify({
-							ssdt: ['spidev1.1'],
-						}),
-					);
-				}
-				return Promise.resolve('');
-			});
-
 			await configFsBackend.initialise();
 			expect(fsUtils.exec).to.be.calledWith('modprobe acpi_configfs');
+
+			// If the module performs this call is because all the prior checks succeeded
```

Comment:
"If the module performs this call is because all the prior checks succeeded" to "If the module performs this call, it's because all the prior checks succeeded"